main.go

```diff
@@ -313,7 +313,7 @@ func main() {
 
 				log.Infof("Test results:")
 				w := tabwriter.NewWriter(os.Stdout, 0, 0, 3, ' ', 0)
-				if _, err := fmt.Fprintln(w, "Model\tOS version\tLocale\tOrientation\tOutcome\t"); err != nil {
+				if _, err := fmt.Fprintln(w, "Model\tOS version\tOrientation\tLocale\tOutcome\t"); err != nil {
```

Comment:
Sadly it will cause the same problem with the output.
![vdt](https://user-images.githubusercontent.com/37296013/49722403-f4df3c80-fc64-11e8-80c5-8f175b427ce6.png)